Sources/BraveShared/BraveStrings.swift

```diff
@@ -1845,6 +1850,73 @@ extension Strings {
         value: "%lld items",
         comment: "%lld is a placeholder and should not be localized. Sometimes items can have no name. So this is a generic title to use")
   }
+  
+  public struct PlaylistFolderSharing {
+    public static let addButtonTitle =
+      NSLocalizedString("playlistFolderSharing.addButtonTitle", tableName: "BraveShared", bundle: .strings,
```

Comment:
I pushed strings in a separate branch, remember about it when rebasing on top off dev

Reply:
Fixed